opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -1,95 +1,51 @@
 use eventheader::{FieldFormat, Level, Opcode};
-use eventheader_dynamic::EventBuilder;
+use eventheader_dynamic::{EventBuilder, EventSet, Provider};
 use opentelemetry::{otel_debug, otel_info};
-use std::borrow::Cow;
-use std::collections::HashMap;
 use std::fmt::Debug;
+use std::sync::Arc;
 
 use opentelemetry::{logs::AnyValue, logs::Severity, Key};
 use opentelemetry_sdk::error::{OTelSdkError, OTelSdkResult};
 use std::{cell::RefCell, str, time::SystemTime};
 
-/// Provider group associated with the user_events exporter
-pub type ProviderGroup = Option<Cow<'static, str>>;
-
 thread_local! { static EBW: RefCell<EventBuilder> = RefCell::new(EventBuilder::new());}
 
-/// Exporter config
-#[derive(Debug)]
-pub struct ExporterConfig {
```

Comment:
This doesn't seem to be related to the caching problem. Maybe remove this in a separate PR?

Reply:
Yes unrelated. It's a cleanup of unused things. I'll see if its easier to do it step by step.